config/env.config.js

```diff
@@ -55,5 +55,60 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
+    this.apply = function(compiler) {
+        compiler.plugin('compilation', function(compilation) {
+            compilation.plugin('html-webpack-plugin-before-html-processing', function(htmlPluginData, callback) {
+                let htmlHead = htmlPluginData.html.match('<head>');
+                if (htmlHead) {
+                    htmlHead = htmlHead.index;
+                    htmlPluginData.html = htmlPluginData.html.slice(0, htmlHead) + '<head>' +
+                    `<script>
```

Comment:
Nit: I think you want the tick mark to be at the top of the previous line.  That way the produced html aligns nicely (or else the script tag has no tab, but everything else does.